src/gen9_avc_encoder.c

```diff
@@ -4016,7 +4017,7 @@ gen9_avc_fei_set_curbe_mbenc(VADriverContextP ctx,
         obj_surface = SURFACE(surface_id);
         if (!obj_surface) {
             WARN_ONCE("Invalid backward reference frame\n");
-            if (gpe_context)
+            if (gpe_context != NULL)
```

Comment:
I think this change is unnecessary .  the two conditions are the same indeed

Reply:
ok, I will delete this update